src/pages/login/index.tsx

```diff
@@ -13,87 +13,108 @@ interface Value {
+      onFinish={auth}
+      aria-label="Some name"
+    >
+      <Form.Item
+        name="email"
+        rules={[{ required: true, message: "Please input your email!" }]}
+        label="Email"
       >
-        <Form.Item
-          name="email"
-          rules={[{ required: true, message: "Please input your email!" }]}
-          label="Email"
-        >
-          <Input
-            prefix={<UserOutlined className="site-form-item-icon" />}
-            placeholder="email"
-          />
-        </Form.Item>
-        <Form.Item
-          name="password"
-          rules={[{ required: true, message: "Please input your Password!" }]}
-          label="Password"
-        >
-          <Input
-            prefix={<LockOutlined className="site-form-item-icon" />}
-            type="password"
-            placeholder="Password"
-          />
+        <Input
+          prefix={<UserOutlined className="site-form-item-icon" />}
+          placeholder="email"
+        />
+      </Form.Item>
+      <Form.Item
+        name="password"
+        rules={[{ required: true, message: "Please input your Password!" }]}
+        label="Password"
+      >
+        <Input
+          prefix={<LockOutlined className="site-form-item-icon" />}
+          type="password"
+          placeholder="Password"
+        />
+      </Form.Item>
+      <Form.Item>
+        <Form.Item name="remember" valuePropName="checked" noStyle>
+          <Checkbox>Remember me</Checkbox>
         </Form.Item>
-        <Form.Item>
-          <Form.Item name="remember" valuePropName="checked" noStyle>
-            <Checkbox>Remember me</Checkbox>
-          </Form.Item>
+        <a className={style.loginForgot} href="" onClick={(e) => showAlert(e)}>
+          Forgot password
+        </a>
+      </Form.Item>
+      <Form.Item>
+        <Button type="primary" htmlType="submit" className={style.loginButton}>
+          Log in
+        </Button>
+        Or{" "}
+        <Link href="/signup">
+          <a href="">register now!</a>
+        </Link>
+      </Form.Item>
+      {statusState.status === "error" && (
+        <h4 className={style.error}>{statusState.message}</h4>
+      )}
+    </Form>
+  );
 
-          <a
-            className={style.loginForgot}
-            href=""
-            onClick={(e) => showAlert(e)}
-          >
-            Forgot password
-          </a>
-        </Form.Item>
-        <Form.Item>
-          <Button
-            type="primary"
-            htmlType="submit"
-            className={style.loginButton}
-          >
-            Log in
-          </Button>
-          Or{" "}
-          <Link href="/signup">
-            <a href="">register now!</a>
-          </Link>
-        </Form.Item>
-        {status === "error" && (
-          <h4 className={style.error}>Oops, something went wrong!</h4>
-        )}
-      </Form>
+  return (
+    <div className={style.loginFormContainer}>
+      {statusState.status === "pending" ? (
```

Comment:
you could just render Spin with loginForm as child node and set `spinning` prop on `Spin` equal to `statusState.status === "pending"`